### PR TITLE
adminActivityLogPage: allow filtering for id #4529

### LINKED ISSUE #4529: adminActivityLogPage: allow filtering for id
as per title

### PR DESCRIPTION
**Pull request by unyoungwax**
Fixes #4529

### CODE REVIEW

**Comment by bamboo3250** [comment]
@unyoungwax Could you update the Reference in the Activity Log Page as well?

image

**Comment by unyoungwax** [comment]
I have added a new label "id" and a new example query similar to the "role" label to the reference.

**Comment by thyageshm** [comment]
@kanghj could we finish this one this release?

File: src/main/webapp/WEB-INF/tags/admin/activity/filterPanel.tag [hunk 1]
```diff
@@ -89,6 +89,7 @@
                                     <li>E.g. time: 1000 (means 1000ms) </li>
                                     <li>E.g. info: Admin Account Management Page Load </li> 
                                     <li>E.g. info: Admin Account Management Page Load, Total, 90 (Use "," to search multiple key strings)</li>
+                                    <li>E.g. id: alice@gmail.com%20160131181745245, charlie@gmail.com%20160201182727734
```
Comment (bamboo3250) [inline src/main/webapp/WEB-INF/tags/admin/activity/filterPanel.tag]: Shouldn't there be a closing tag &lt;/lt&gt; here?

**Comment by thyageshm** [comment]
@unyoungwax please try and finish this one soon. I believe it is almost ready for merging

**Comment by unyoungwax** [comment]
I have added the closing tag. It is ready for review.

**Comment by bamboo3250** [comment]
@damithc This PR is ready for your review.

**Comment by thyageshm** [comment]
@unyoungwax please merge this one soon.

**Comment by damithc** [comment]
I think @bamboo3250 needs to merge this one.

**Comment by bamboo3250** [comment]
Sorry, I didn't notice this PR was approved. I'll merge it tonight.

**Pull request merged**